### PR TITLE
Feat/customers

### PR DESCRIPTION
**Pull request by BdsOliveira**

### CODE REVIEW

File: src/views/Customers/CustomerForm.vue [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+          <v-divider :thickness="4" class="border-opacity-100" color="info"></v-divider>
+          <span class="d-flex justify-space-between bg-light_grey px-3 mb-2">
+            <span class="">Customer address: #{{ key + 1 }}</span>
+            <span class="cursorPointer" @click="removeAddress(key)">
+              <v-icon color="danger" icon="mdi-close"></v-icon>
+            </span>
+          </span>
+          <v-row>
+            <v-col cols="4">
+              <v-text-field v-model="address.phone" label="Customer phone"></v-text-field>
```
Comment (danieldsf) [inline src/views/Customers/CustomerForm.vue]: This time let's use "Address/Customer phone"

File: src/views/Customers/CustomerForm.vue [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+            </v-col>
+            <v-col cols="4">
+              <v-text-field v-model="address.country" label="Address Country"></v-text-field>
+            </v-col>
+            <v-col cols="4">
+              <v-text-field v-model="address.zipcode" label="Customer Zipcode"></v-text-field>
```
Comment (danieldsf) [inline src/views/Customers/CustomerForm.vue]: Address Zipcode

File: src/views/Customers/CustomerForm.vue [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+            </v-col>
+          </v-row>
+          <v-row>
+            <v-col cols="4">
+              <v-text-field v-model="address.state" label="Customer phone"></v-text-field>
```
Comment (danieldsf) [inline src/views/Customers/CustomerForm.vue]: State?

File: src/views/Customers/CustomerForm.vue [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+            </v-col>
+            <v-col cols="4">
+              <v-text-field v-model="address.city" label="Address Country"></v-text-field>
```
Comment (danieldsf) [inline src/views/Customers/CustomerForm.vue]: City?

**Review by danieldsf — CHANGES_REQUESTED** [review]
Looking good, just a few verbiage issues.

File: src/views/Customers/CustomerForm.vue (hunk 1 above)
Comment (BdsOliveira) [reply src/views/Customers/CustomerForm.vue]: Fixed

File: src/views/Customers/CustomerForm.vue (hunk 2 above)
Comment (BdsOliveira) [reply src/views/Customers/CustomerForm.vue]: Fixed.

File: src/views/Customers/CustomerForm.vue (hunk 3 above)
Comment (BdsOliveira) [reply src/views/Customers/CustomerForm.vue]: CTRL+C -> CTRL + V 😵‍💫 
Fixed.

File: src/views/Customers/CustomerForm.vue (hunk 4 above)
Comment (BdsOliveira) [reply src/views/Customers/CustomerForm.vue]: Fixed.

**Review by danieldsf — APPROVED** [review]

**Pull request merged**